### ISSUE TITLE
Default number of rows?

### ISSUE THREAD

**Issue by cibulka** [rpearce/react-expanding-textarea#14]
Is there a recommended way how to set the minimum number of rows for `react-expanding-textarea`? Settings attribute `rows` does not seem to work.

```
import Textarea from 'react-expanding-textarea';
// render
<Textarea rows="3" />
```

Thanks in advance!

**Comment by rpearce** [comment]
I'm not 100% sure I want to add that functionality, but I'm more than willing to chat about it.

Let's see what might go into it:
* `line-height` * `n` where `n` is `minRows`
* add in vertical `padding` calculation to get final number
* set the height to `Math.max(minHeight, height)` where `minHeight` is the result of the calculation and the `height` is what it would be

Does that sound right? What am I missing?

**Comment by cibulka** [comment]
Pretty much.

I like that your library just works out of the box - with no additional CSS configuration. As HTML attributes `rows` do what I'm after, it would be great, if your library took that into account.

**Comment by rpearce** [comment]
I dig it. I'll see what I can do when I get some time soon

**Comment by rpearce** [comment]
Update: taking a stab at this now...

**Comment by rpearce** [comment]
@cibulka Can you please review https://github.com/rpearce/react-expanding-textarea/pull/17?

**Comment by rpearce** [comment]
(pinging @cibulka again)

**Comment by cibulka** [comment]
Sorry for the delay. Looks & works great though, I very much approve!

**Comment by rpearce** [comment]
@cibulka This is now published under version `0.2.0`. Thanks for contributing!
